go-selinux/rchcon.go

```diff
@@ -12,7 +12,22 @@ import (
 )
 
 func rchcon(fpath, label string) error {
+	slowMode := false
```

Comment:
Perhaps a silly question, but what does "slowMode" mean here? Looking at the changes, it seems that `slowMode` actually enables a "fast path" (check the label, and we're done), but the name of this variable feels like it has the reverse meaning 😅  

Or is reading the label (`lFileLabel()`) slower than setting it (` lSetFileLabel()`) ?